azure/hub/examples/advanced/main.tf

```diff
@@ -58,11 +59,29 @@ module "hub" {
   }
 
   network_watcher_config = {
-    name                = "NetworkWatcher_UKSouth-${local.resource_prefix}"
+    name                = "nw_uks-${local.resource_prefix}"
     resource_group_name = "rg-nw-${local.resource_prefix}"
   }
 }
 
+module "firewall-policy" {
+  source = "git::https://github.com/ans-coe/terraform-modules.git//azure/firewall-policy/?ref=079d443bdb640628368408e547acfbf8bf18f9b7"
```

Comment:
use a local path for this

Reply:
Changed back to local
was changed for testing